cmd/kogito/command/install/trusty.go

```diff
@@ -0,0 +1,221 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
```

Comment:
Hi @radtriste , I've added some tests. Since the trusty service has basically the same dependencies of the data-index (infinispan + kafka) I did not reinvent the wheel. What if I create a `common.go` file to refactor a bit the tests that are the same? We might refactor the tests that needs infinispan arguments and so on, wdyt?

Reply:
I would rather keep each one separate for the unit tests.
@ricardozanini wdyt ?

Reply:
@r00ta they should be kept on their files. What you can do is create support functions in a `cmd/kogito/test` package and re use them across test cases.